### PR TITLE
Doc changes for issue 632

### PR DESCRIPTION
**Pull request by micgibso**
Signed-off-by: micgibso <micgibso@uk.ibm.com>

### CODE REVIEW

**Comment by micgibso** [comment]
@tetchel Thanks Tim, couple of questions:

_3. We should add the Remove Project command as follows:_
Does this need adding to the Eclipse topic, mdt-eclipse-managingprojects.md, too? It's not there atm. I'm guessing it ought to be? 

_There's a weirdly large gap of whitespace between the Commands and Logs sections._
Not seeing that in either Firefox, Chrome, or Edge.  Are you still seeing it?

**Comment by tetchel** [comment]
Eclipse does have 'remove project', [here](https://www.eclipse.org/codewind/mdteclipsemanagingprojects.html#project-actions)

File: docs/_documentations/mdt-vsc-commands-project.md [hunk 1]
```diff
@@ -23,21 +23,25 @@ Right-click a project in the **Codewind view** to see most project commands. All
 
 - **Open app**: Open the project application root endpoint in the system browser. The project must be in the *Running* or *Debugging* state, or the application server won't be available.
 
-- **Open application monitor**: Open the Codewind **Application Monitor** page for this project in the system browser.
+- **Show Project Overview**: Open an editor tab that displays all of the Codewind information for the project. From this page, you can build, disable, or delete the project, and you can change the project auto build setting. This page is the only place in the extension where you can delete a project. If you delete a project, you remove it from both Codewind and from your file system.
```
Comment (tetchel) [inline docs/_documentations/mdt-vsc-commands-project.md]: oh, just noticed! this has some outdated info.

```
This page is the only place in the extension where you can delete a project. If you delete a project, you remove it from both Codewind and from your file system. 
```
Please remove these two sentences

File: docs/_documentations/mdt-vsc-commands-project.md [hunk 2]
```diff
@@ -23,21 +23,25 @@ Right-click a project in the **Codewind view** to see most project commands. All
...
 - **Build**: Manually request an application build for this project. If the project has auto build enabled, this command should not be necessary because builds are triggered automatically with a code change. This command is also available in the project info page.
 This command is equivalent to clicking the **Build** button in the Codewind **Overview** page.
 
 - **Toggle auto build**: Enable or disable auto build for the project. This command is also available in the project info page. This command is equivalent to clicking the **Auto Build** toggle in the Codewind **Overview** page.
 
-- **Open container shell**: Open a shell, either *bash* or *sh*, in the project application container with *docker exec*. The project must have a container running. The VS Code integrated terminal needs access to your *PATH* environment variable so that it can run the *docker* command.
-
 - **Enable or disable project**: Enable or disable the project. This command is also available in the project info page.<br>
 
+- **Remove project**: Removes the project from Codewind. The project folder can optionally be deleted from disk after removal.
```
Comment (tetchel) [inline docs/_documentations/mdt-vsc-commands-project.md]: Also, for consistency with Disable/Enable and Build, let's add:

`This command is also available in the project info page.` 

to this one.

**Comment by tetchel** [comment]
I realized the command is called Open Project Overview, but we refer to it as "the project info page". 

Can we replace "project info page" with "project overview" ?

File: docs/_documentations/mdt-vsc-commands-project.md (hunk 1 above)
Comment (micgibso) [reply docs/_documentations/mdt-vsc-commands-project.md]: Removed.

File: docs/_documentations/mdt-vsc-commands-project.md (hunk 2 above)
Comment (micgibso) [reply docs/_documentations/mdt-vsc-commands-project.md]: Added to Remove Project

Comment (micgibso) [reply docs/_documentations/mdt-vsc-commands-project.md]: Added, and marked resolved.

**Comment by micgibso** [comment]
@tetchel No worries, all changes ready to check in except the last one. Under `Build` we have the following: 

`This command is also available in the project info page.
This command is equivalent to clicking the **Build** button in the Codewind **Overview** page.`

Is this talking about the same page, or two different pages? Is it the case that the `project overview page` is different to the `Codewind **Overview** page`? 

Welcoming Mel's and your work to cut through this and standardise the terminology. ;)

**Comment by micgibso** [comment]
@tetchel Hi Tim, no probs, all addressed. One question, is the option in the commands definitely **Show Project Overview** ?  It's just contrary to all the other options under `Commands` because they all start  `Open`.

**Comment by tetchel** [comment]
good catch, it is indeed `Open Project Overview`

**Comment by micgibso** [comment]
@tetchel Thanks, changed Show... to Open...

File: docs/_documentations/mdt-vsc-commands-project.md [hunk 3]
```diff
@@ -23,20 +23,23 @@ Right-click a project in the **Codewind view** to see most project commands. All
...
+- **Open Performance Dashboard**: Opens the performance dashboard in the browser. This action is available only when the application is running or debugging.
 
-- **Toggle auto build**: Enable or disable auto build for the project. This command is also available in the project info page. This command is equivalent to clicking the **Auto Build** toggle in the Codewind **Overview** page.
+- **Add Project to Workspace**: Add this project's folder on your local disk to your VS Code workspace.
 
-- **Open container shell**: Open a shell, either *bash* or *sh*, in the project application container with *docker exec*. The project must have a container running. The VS Code integrated terminal needs access to your *PATH* environment variable so that it can run the *docker* command.
+- **Build**: Manually request an application build for this project. If the project has auto build enabled, this command should not be necessary because builds are triggered automatically with a code change. This command is also available in the project i page. This command is equivalent to clicking the **Build** button in the Codewind **Overview** page.
```
Comment (tetchel) [inline docs/_documentations/mdt-vsc-commands-project.md]: `project i page` -> project overview page

File: docs/_documentations/mdt-vsc-commands-project.md [hunk 4]
```diff
@@ -23,20 +23,23 @@ Right-click a project in the **Codewind view** to see most project commands. All
...
 
-- **Enable or disable project**: Enable or disable the project. This command is also available in the project info page.<br>
+- **Toggle auto build**: Enable or disable auto build for the project. This command is also available in the project overview. This command is equivalent to clicking the **Auto Build** toggle in the Codewind **Overview** page.
```
Comment (tetchel) [inline docs/_documentations/mdt-vsc-commands-project.md]: Remove 

This command is equivalent to clicking the **Auto Build** toggle in the Codewind **Overview** page.

It has the same info as the same as the previous sentence

File: docs/_documentations/mdt-vsc-commands-project.md (hunk 3 above)
Comment (micgibso) [reply docs/_documentations/mdt-vsc-commands-project.md]: Good spot, corrected.

File: docs/_documentations/mdt-vsc-commands-project.md (hunk 4 above)
Comment (micgibso) [reply docs/_documentations/mdt-vsc-commands-project.md]: Removed.

**Comment by micgibso** [comment]
@tetchel All changes made, thanks.

**Pull request merged**